backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/scheduling/SchedulingManager.java

```diff
@@ -697,6 +697,8 @@ private void refreshCachedPendingValues(List<VDS> vdsList) {
     }
 
     private List<List<VM>> groupVms(List<VM> vms, SchedulingContext context) {
+        // Sorting the VMs to get better dedicated CPU pinning.
```

Comment:
this comment doesn't add much value without explaining what "better" means..

Reply:
let's just drop it.